.env.example

```diff
@@ -1,5 +1,5 @@
-# Base Path Openshift: businesses/create Firebase:
-VUE_APP_PATH=businesses/create
+# Base Path Openshift: /businesses/create Firebase: /
+VUE_APP_PATH=/businesses/create
```

Comment:
Are quotes needed?

Reply:
Doesn't matter, you can add or remove it.

Reply:
Then let's keep it consistent with the other values in here.

Also in the fine below please.

Reply:
Ok, I will add the quote for all places.